core/src/cluster_info.rs

```diff
@@ -101,6 +107,8 @@ pub struct ClusterInfo {
     pub(crate) keypair: Arc<Keypair>,
     /// The network entrypoint
     entrypoint: Option<ContactInfo>,
+
+    outbound_budget: (usize, u64),
```

Comment:
Can you please add a comment for the tuple values, it's not intuitive from the types what either value is used for

Reply:
Actually just make it a struct?